src/main/java/net/atomique/ksar/Main.java

```diff
@@ -18,13 +26,22 @@
 public class Main {
 
   private static final Logger log = LoggerFactory.getLogger(Main.class);
-
+  private static boolean gui = true;
   static Config config = null;
   static GlobalOptions globaloptions = null;
   static ResourceBundle resource = ResourceBundle.getBundle("net/atomique/ksar/Language/Message");
 
   public static void usage() {
-    show_version();
+    log.info("-input : input sar statistics file \n");
+    log.info("-n : toggles nongui mode for exporting\n");
+    log.info("-outputCSV : location of parsed CSV file\n");
+    log.info("-outputPDF : location of output pdf file\n");
+    log.info("-outputIMG : prefix for output images\n");
+    log.info("-width : width for output png charts\n");
+    log.info("-heigth : heigth for output png charts");
```

Comment:
`height` should be spelled as `height`.
Of course naming is hard, however it looks odd to have `width` and `height` options dedicated to `png`.
What if we require pdf page width/height?

I think `imgWidth` would be better.

PS. Something like https://github.com/airlift/airline or https://github.com/tatsuhiro-t/argparse4j might make sense to parse the options